phoenixRest/views/friend_request/instance.py

```diff
@@ -0,0 +1,73 @@
+from pyramid.view import view_config, view_defaults
+
+from pyramid.httpexceptions import HTTPNotFound
+
+from pyramid.authorization import Allow, Deny
+from phoenixRest.roles import ADMIN
+
+from phoenixRest.models.core.friendship import Friendship
+
+from phoenixRest.utils import validateUuidAndQuery
+from phoenixRest.utils import validate
+
+from sqlalchemy import and_, or_, extract
+
+from datetime import datetime
+
+class FriendRequestInstanceResource(object):
+    def __acl__(self):
+        source_user_uuid = self.friendRequestInstance.source_user.uuid
+        recipient_user_uuid = self.friendRequestInstance.recipient_user.uuid
+        acl = [
+            (Allow, ADMIN, "view")
+            (Allow, f"{source_user_uuid}", "view"),
```

Comment:
What about not using a format string? you can wrap `self.friendRequestInstance.source_user.uuid` in `str()` to convert the uuid to a string